Lib/asyncio/windows_events.py

```diff
@@ -834,7 +834,7 @@ def close(self):
             return
 
         # Cancel remaining registered operations.
-        for address, (fut, ov, obj, callback) in list(self._cache.items()):
+        for fut, ov, obj, callback in self._cache.values():
```

Comment:
`ov, obj, callback` are unused as well, but I've decided to keep them, because that's how other places do.

Reply:
I think list was necessary here to make a copy of the cache before iteration so that if any thread makes any change to the cache then it won't cause a RuntimeError. Moreover IIRC list on dict items is an atomic operation as it calls a c function.

Reply:
`fut.cancel()` can have a registered callback with a side-effect that modifies `self._cache` dict.
The unrecoverable failure in `loop.close()` is not desirable.
`self._cache.items()` can be replaced with `self._cache.values()` safely, sure.

Reply:
Thanks! TIL 🙂